hackathon_site/event/jinja2/event/landing.html

```diff
@@ -2,9 +2,9 @@
 
 {% block head_extends %}
```

Comment:
Change the title of the ticket to the template I sent in the Slack Webmaster channel. 

And the reason why you are failing the Github check is because you didn't run black (the python file formatted) before you push your code. You should always run that before you commit/push.

Reply:
Thank you! The title has been changed, and the checks are all passing now